### PR TITLE
add command to set WEBOTS_SAFE_MODE in windows

### PR DESCRIPTION
**Pull request by Behrouz-m**
**Documentation**
If this pull-request changes the doc, add the link to the related page, including the `?version=BRANCH_NAME`, such as:
https://cyberbotics.com/doc/guide/starting-webots?version=R2021a

### CODE REVIEW

File: docs/guide/starting-webots.md [hunk 1]
```diff
@@ -128,10 +128,16 @@ This action may cause a new crash.
 
 #### On Windows
 
+Using  GUI:
```
Comment (omichel) [inline docs/guide/starting-webots.md:131]:
```suggestion
From the Windows graphical user interface:
```

File: docs/guide/starting-webots.md [hunk 2]
```diff
@@ -128,10 +128,16 @@ This action may cause a new crash.
...
 1. Open the `Environment Variables` system dialog box. To do so, look for "environment variable" in the `search bar` of the Windows `start menu`, click on `Edit the system environment variables`, this will open up the `System Properties` dialog box to the `Advanced` tab. Click on the `Environment Variables` button at the bottom.
 2. Add a new `WEBOTS_SAFE_MODE` user environment variable. To do so, in the `user variables` panel, click on the `New` button and add a `New User Variable` named `WEBOTS_SAFE_MODE` with a value of `true`.
 3. Start Webots as usual.
 
+Using cmd:
```
Comment (omichel) [inline docs/guide/starting-webots.md:136]:
```suggestion
From the `cmd` command prompt:
```

**Review by omichel — CHANGES_REQUESTED** [review]
Thank you for this contribution. I only have a couple of styling requests.

**Comment by Behrouz-m** [comment]
@omichel please check it. thanks for the suggestion.

**Review by omichel — APPROVED** [review]
Thank you!

**Comment by omichel** [comment]
@Behrouz-m: I just sent you an invite to become a member of the Committers group. Once accepted, you should be able to merge this PR.

**Comment by Behrouz-m** [comment]
> 
> 
> @Behrouz-m: I just sent you an invite to become a member of the Committers group. Once accepted, you should be able to merge this PR.

Thank you. It was unexpected :)
I am happy to be a member of your team.

**Comment by omichel** [comment]
> Thank you. It was unexpected :)
I am happy to be a member of your team.

You are welcome!

**Pull request merged**